.github/workflows/compilation_on_android_ubuntu.yml

```diff
@@ -448,7 +448,7 @@ jobs:
         run: echo "::error::can not get prebuilt llvm libraries" && exit 1
 
       - name: run tests
-        run: ./test_wamr.sh ${{ matrix.test_option }} -t ${{ matrix.running_mode }}
+        run: ./core/iwasm/libraries/lib-socket/test/build.sh && ./test_wamr.sh ${{ matrix.test_option }} -t ${{ matrix.running_mode }}
```

Comment:
Should be `../../core/iwasm/libraries/lib-socket/test/build.sh`, since the working-directory is `./tests/wamr-test-suites`

Reply:
also, you might want to move build to a separate step so it's easier to spot in CI what actually failed (build or test)